test/functional/NativeTest/playlist.xml

```diff
@@ -104,6 +104,38 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<subset>11+</subset>
 		</subsets>
 	</test>
+	<test>
+		<testCaseName>jniargtestssystemlink</testCaseName>
+		<variations>
+			<variation>-Xint</variation>
+			<variation>-Xjit:count=0</variation>
+			<variation>-Xjit:count=1,disableAsyncCompilation</variation>
+		</variations>
+    <platformRequirements>os.zos,bits.31</platformRequirements>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
+	$(JAVA_COMMAND) $(JVM_OPTIONS) \
+	-cp $(Q)$(TEST_RESROOT)$(D)jniargtests$(D)jniargtests.jar$(Q) JniArgTests jniargtestssystemlink; \
+	$(TEST_STATUS)
+		</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<types>
+			<type>native</type>
+		</types>
+		<impls>
+			<impl>openj9</impl>
+			<impl>ibm</impl>
+		</impls>
+		<subsets>
+			<subset>8</subset>
+			<subset>9</subset>
+			<subset>11+</subset>
+		</subsets>
```

Comment:
Please remove this useless `<subsets>` element.

Reply:
Fixed in aa851c4 (see [force push](https://github.com/eclipse/openj9/compare/5f938ef88bb3f8d33239c8a3c05657370baa457e..ab713cac6f91859325f0e62136936b0bb7ddbec8)). I also took this opportunity to fix all similar `<subsets>` in the playlist.